aim/agent/aid/universes/aci/converter.py

```diff
@@ -1048,6 +1048,45 @@ def bgp_as_id_converter(object_dict, otype, helper, to_aim=True):
     'vmmRsVswitchExporterPol': [{
         'resource': resource.VmmRelationToExporterPol,
     }],
+    'spanVSrcGrp': [{
+        'resource': resource.SpanVsourceGroup,
+    }],
+    'spanVSrc': [{
+        'resource': resource.SpanVsource,
+    }],
+    'spanVDestGrp': [{
+        'resource': resource.SpanVdestGroup,
+    }],
+    'spanVDest': [{
+        'resource': resource.SpanVdest,
+    }],
+    'spanVEpgSummary': [{
+        'resource': resource.SpanVepgSummary,
+    }],
+    'spanRsSrcToVPort': [{
+        'resource': resource.SpanSrcVport,
+    }],
+    'infraAccBndlGrp': [{
+        'resource': resource.InfraAccBundleGroup,
+    }],
+    'infraAccPortGrp': [{
+        'resource': resource.InfraAccPortGroup,
+    }],
+    'infraRsSpanVSrcGrp': [{
```

Comment:
For infraRsSpanVSrcGrp, we associate the VsrcGrp to a vpc interface (InfraAccBundleGroup). Would the user has to configure/update InfraAccBundleGroup since infraRsSpanVSrcGrp will be a member of this class?